microsoft/microsoft-ui-xaml
# Update gamepad helper

## Description

This updates the GamePad helper with the hope to improve the reliability for tests that inject GamePad input.

General idea was: Press button, wait 50 milliseconds, lift button.

New idea: Press button, wait for testframe to report input was received, wait for idle and lift button again.
If the testframe doesn't have the checkbox to report it received the input, we fallback to the old behavior.

## Motivation and Context




## How Has This Been Tested?

Ran tests multiple times, though they were reliable on my machine before this change too.

## How should we test this?
I think we should run CI with the changes, if everything goes fine, we can try reenabling tests and run the CI multiple times to determine if those tests are reliable enough.

The tests that are currently disabled due to gamepad issues are:

- RatingControlTests.GamepadTest
- TreeViewTests.TreeViewMultiSelectGamepadTest_ContentMode
- RadioButtonsTests.GamepadCanEscapeAndDoesNotSelectWithFocus

## Screenshots (if appropriate):
